cmd/run.go

```diff
@@ -46,21 +46,26 @@ func buildRunCommand() *cobra.Command {
 
 			// load TTP and process argument values
 			// based on the TTPs argument value specifications
-			c := blocks.TTPExecutionConfig{
-				Repo: foundRepo,
+			ttpCfg.Repo = foundRepo
+
+			// Set the --no-cleanup value if provided
+			ttpCfg.NoCleanup, err = cmd.Flags().GetBool("no-cleanup")
```

Comment:
I think this line is redundant with the line below:

```
	runCmd.PersistentFlags().BoolVar(&ttpCfg.NoCleanup, "no-cleanup", false, "Disable cleanup (useful for debugging and daisy-chaining TTPs)")
```

Your removal of `c` so that `ttpCfg is used correctly throughout should be sufficient to fix the issue on its own

Reply:
You are absolutely right, good catch. Added the appropriate fixes to this PR